UI/setup/select_coa.html

```diff
@@ -24,11 +24,19 @@
            label = text('Chart of accounts')
     };
 ELSE;
+    lang = '';
+    IF languages;
+        lang = languages.pop;
+        IF lang.search('-');
+            lang = lang.split('-').1;
+        END;
+    END;
```

Comment:
I agree, that HTTP::AcceptLanguage will do the job, but it then makes our code dependant on being delivered via an unmolested browser <==> server interface.
There are security helpers out there (especially in today's world of Metadata retention) that strip information like this from the Headers.
Hence my suggestion to also include it in the JS
We may want to compare the 2 results and prompt the user to select if they differ